src/components/graypane.ts

```diff
@@ -29,10 +29,11 @@ export class Graypane {
       .getWin()
       .document.createElement('swg-popup-background');
 
+    this.fadeBackground_.setAttribute('role', 'button');
```

Comment:
can we add a comment that actually setting the role button attribute is to block clicks through the pane?